src/views/Settings.vue

```diff
@@ -58,14 +58,15 @@ export default defineComponent({
       uploadProducts: 'product/getUploadProducts'
     })
   },
+  mounted(){
+    console.log(this.currentFacility)
+  },
```

Comment:
Remove unwanted code

Reply:
Removed unwanted code Sir.
Thank you sir for reviewing my code.